bin/sdl.cc

```diff
@@ -77,13 +77,13 @@ int main(int argc, char** argv)
 
     // A default value one
     if (ana.input_file_list_tstring.EqualTo("muonGun"))
-        ana.input_file_list_tstring = "/nfs-7/userdata/phchang/trackingNtuple/trackingNtuple_100_pt0p5_2p0.root";
+        ana.input_file_list_tstring = "/data2/segmentlinking/trackingNtuple_100_pt0p5_2p0.root";
```

Comment:
on phi3 these files are in `/data2/segmentlinking/trackingNtuple/CMSSW_10_4_0_patch1-orig`